copc/lidar-las-to-copc.ipynb

```diff
@@ -0,0 +1,660 @@
+{
```

Comment:
<p>I'm not sure the print statement is the best way to explore the PDAL object. Can we do a more python oriented method showing maybe the top level or top 2 levels of keys.</p><p> </p><p>Also explain if someone wants to work with the data what would they do next, read the <code>array values</code>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cloudnativegeo/cloud-optimized-geospatial-formats-guide/pull/109/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>